src/components/features/ProductSearch/ProductSearch.js

```diff
@@ -10,6 +10,26 @@ const ProductSearch = () => (
   <form action='' className={styles.root}>
     <div className={styles.category}>
       <FontAwesomeIcon className={styles.icon} icon={faListUl} />
+      <ul>
```

Comment:
Wychodź z `mastera` bo znowu ciągną się zmiany z innego zadania i to zadanie też nie bedzie mogło być zaakceptowane dopóki task 10 nie bedzie zmergowany. Dodatkowo po poprawkach taska 10 tutaj będą konflikty.